docs/libu64.md

```diff
@@ -0,0 +1,73 @@
+# libu64
+
+This document explains the reasoning behind discovering libu64 files.
```

Comment:
"reasoning behind discovering" -> "reasoning behind organizing" or something? We didn't really "decide" to discover them